lib/tests/test_ecl_grid.cpp

```diff
@@ -0,0 +1,321 @@
+#include <catch2/catch.hpp>
+#include <ert/ecl/ecl_grid.hpp>
+#include <ert/ecl/ecl_kw.hpp>
+#include <ert/ecl/ecl_kw_magic.hpp>
+#include <vector>
+
+#include "tmpdir.hpp"
+
+using namespace std;
```

Comment:
Don't `using namespace std;`. Prefer either writing out `std::` or importing each symbol separately using `using std::vector;`

Reply:
Fixed by not doing `using namespace std;`